lib/data/locationConstraintParser.js

```diff
@@ -0,0 +1,41 @@
+import file from './file/backend';
+import inMemory from './in_memory/backend';
+import Sproxy from 'sproxydclient';
+
+import config from '../Config';
+
+const clients = {};
+
+export default function parseLC() {
```

Comment:
This function does not seem to parse anything (except some JSON internally) but to initialize a global table of clients, so the name seems a bit off-topic. (same comment for where it's used)